index.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var app = express();
+var metrics = require('node-monitor-ui');
+
+const PORT = process.env.PORT || 3000;
+var USE_GCI = process.env.USE_GCI || 'FALSE';
+var MSG_SIZE = process.env.MSG_SIZE || 12;
+var WINDOW_SIZE = process.env.WINDOW_SIZE || 12;
```

Comment:
Please change the or to 0

Reply:
MSG_SIZE = 1024 and WINDOW_SIZE = 1? Or not?

Reply:
MSG_SIZE = 1024 --> correct

Default value of **WINDOW_SIZE = 0**